EIPS/eip-2357.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2357
+title: Total Difficulty in Block Header
+author: Tomasz K. Stanczak (@tkstanczak)
+discussions-to: https://ethereum-magicians.org/t/eip-2357-replace-difficulty-with-total-difficulty-in-block-headers/4233
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-29
+---
+
+## Simple Summary
```

Comment:
No need for both `Simple Summary` and `Abstract`. This requirement has been abolished a while ago.